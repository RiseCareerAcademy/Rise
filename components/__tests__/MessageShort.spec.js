import 'react-native';
import React from 'react';
import shortM from '../MessageShort';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const props = {
        message: 'MyMessage',
        fromUser: 'FromUser',
        sentMessage: 'sentMessage',
    };
  const tree = renderer.create(<shortM {...props}>Snapshot test!</shortM>).toJSON();

  expect(tree).toMatchSnapshot();
});
