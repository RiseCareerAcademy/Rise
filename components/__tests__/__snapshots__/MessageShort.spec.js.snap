// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<shortM
  fromUser="FromUser"
  message="MyMessage"
  sentMessage="sentMessage"
>
  Snapshot test!
</shortM>
`;
